app/src/components/GrantList.vue

```diff
@@ -88,13 +95,22 @@ export default defineComponent({
   },
   setup(props) {
     const { addToCart, isInCart, removeFromCart } = useCartStore();
+    const grantList = computed(() =>
+      grantIdList.value.length > 0 ? createCustomGrantList(props.grants) : props.grants
+    );
+
+    onMounted(async () => {
+      const url =
+        'https://raw.githubusercontent.com/dcgtc/dgrants/top-level-grant-filter/app/src/CustomGrantIndexList.json';
```

Comment:
@phutchins @apbendi Any idea where we want to host these JSON files? (plural, since we'll need one per network)

Reply:
also, quick note to self: if we choose to do it this way, need to remember to change branch in url from 'top-level-grant-filter' to 'main'

Reply:
We could also format the JSON file by chain ID so we only need one file, e.g.

```json
{
  "4": [0,2,7],
  "137": [1,5,6, 9],
}
```

and the app would use `DGRANTS_CHAIN_ID` to find the correct array

Reply:
+1 to that idea - should we wait for input? maybe bring this up in tomorrow's standup